Emergence/Contexts/Presenting Shows/ShowViewController.swift

```diff
@@ -16,43 +19,65 @@ class ShowViewController: UIViewController {
         }
 
         let network = appVC.context.network
-        if let shows = network.root.transitions["show"] {
-            let attributes = ["id": "4ea19ee97bab1a0001001908"]
-            network.request(shows, parameters: attributes).mapTransitionToObject(Show).subscribeNext { showObject in
+
+        let showInfo = ArtsyAPI.ShowInfo(showID: "4ea19ee97bab1a0001001908")
+        network.request(showInfo).mapSuccessfulHTTPToObject(Show).subscribe(next: { showObject in
                 let show = showObject as! Show
-                print(show.name)
-            }
-        }
+                self.showDidLoad(show)
+
+            }, error: { error in
+                print("ERROROR \(error)")
+            }, completed: nil, disposed: nil)
+
+    }
+
+    func showDidLoad(show: Show) {
+        showTitleLabel.text = show.name
     }
 }
 
 enum ORMError : ErrorType {
     case ORMNoRepresentor
+    case ORMNotSuccessfulHTTP
     case ORMNoData
     case ORMCouldNotMakeObjectError
 }
 
 extension Observable {
-    func mapTransitionToObject(classType: Decodable.Type) -> Observable<Decodable> {
+    func mapSuccessfulHTTPToObject(classType: Decodable.Type) -> Observable<Decodable> {
 
         func resultFromJSON(object:[String: AnyObject], classType: Decodable.Type) -> Decodable? {
             return classType.init(json: object)
         }
 
         return map { representor in
-            guard let rep = representor as? Representor<HTTPTransition> else {
+            guard let response = representor as? MoyaResponse else {
                 throw ORMError.ORMNoRepresentor
             }
 
-            if rep.attributes.count == 0 {
-                throw ORMError.ORMNoData
+            // Allow successful HTTP codes
+            if ((200...209) ~= response.statusCode) == false {
```

Comment:
There _should_ be existing data processing methods for RxSwift like the ones for RAC in the latest version of Moya. If they'd help.